pkg/resources/stream_on_table_acceptance_test.go

```diff
@@ -476,33 +476,11 @@ func TestAcc_StreamOnTable_RecreateWhenStale(t *testing.T) {
 					})),
 				),
 			},
-			// set schema parameters to bigger values ensuring that the stream is not stale
-			{
-				PreConfig: func() {
```

Comment:
Why was it removed? (same in other file)

Reply:
Because before there was one test for handling staleness. As a result of https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3129#discussion_r1801940817, it was split up, and here I removed this step not to test the same thing twice - see also https://github.com/Snowflake-Labs/terraform-provider-snowflake/pull/3129#discussion_r1806639261